assets/css/translation.css

```diff
@@ -1,23 +1,50 @@
-html[dir='ltr'] .languageName {
-    padding-left: .5em;
+.langSelectGroup {
+    font-size: 10px;
 }
 
-html[dir='rtl'] .languageName {
-    padding-right: .5em;
+html[dir='ltr'] .langSelectGroup > *:not(*:last-child) {
+    margin-right: 3px;
 }
 
-html[dir='ltr'] .languageVariant {
-    font-size: 12px;
-    padding-left: 2.5em;
+html[dir='rtl'] .langSelectGroup > *:not(*:first-child) {
+    margin-right: 3px;
 }
 
-html[dir='rtl'] .languageVariant {
-    font-size: 12px;
-    padding-right: 2.5em;
+/* Monkeypatch for a bug with grid,
+   they don't fit in one line otherwise */
+.dstLangParent {
+    width: calc(50% - 5px);
+}
+
+html[dir='rtl'] #srcLang1,
+html[dir='rtl'] #dstLang1 {
+	border-radius: 0 .2rem .2rem 0;
```

Comment:
The spacing here is tab instead of spaces like everywhere else.

Reply:
That's my fault. I will change it. :)